packages/gafl-webapp-service/src/pages/contact/name/__tests__/route.spec.js

```diff
@@ -1,23 +1,45 @@
 import { getData } from '../route'
 
-describe('name > route', () => {
-  const mockTransactionCacheGet = jest.fn()
+const getMockPermission = ({ isLicenceForYou }) => ({
```

Comment:
If you default this to true, you can eliminate a bit of code in a couple of tests:
```
const getMockPermission = ({ isLicenceForYou } = { isLicenceForYou: true }) => ({
```